__tests__/index.spec.ts

```diff
@@ -37,6 +37,10 @@ const getGithubMock = () =>
   });
 
 describe('use-herald-action', () => {
+  const owner = 'gagoar';
+  const repo = 'example_repo';
+  const sha = 'ec26c3e57ca3a959ca5aad62de7213c562f8c821';
+
```

Comment:
why do you need these available here and not within the beforeEach? are these used elsewhere?

Reply:
because i need them on two tests, so i don't need to declare them twice on each test